mypy/checkexpr.py

```diff
@@ -2429,6 +2438,19 @@ def narrow_type_from_binder(self, expr: Expression, known_type: Type) -> Type:
         return known_type
 
 
+def has_any_type(t: Type) -> bool:
+    """Whether t contains an Any type"""
+    return t.accept(HasAnyType())
+
+
+class HasAnyType(TypeQuery[bool]):
```

Comment:
Elsewhere in the file `TypeQuery` is referred to as the partially qualified `types.TypeQuery`.  I'd do that here for consistency.